tensorflow/core/framework/allocator.h

```diff
@@ -152,6 +152,7 @@ class Allocator {
   // allocated by this allocator.
   virtual size_t RequestedSize(void* ptr) {
     CHECK(false) << "allocator doesn't track sizes";
+    return size_t(0);
```

Comment:
it is only needed because msvc fails debug builds that the function has no return value